weave/integrations/openai/openai_sdk.py

```diff
@@ -15,6 +16,13 @@
     _add_accumulator,
     _default_on_input_handler,
 )
+from weave.utils.stream_metrics import (
+    WEAVE_STREAM_START_TIME,
```

Comment:
nit & totally not required but for style consistency you could add a function or two which perform the operations you do with this const to keep the logic flow a bit cleaner in here. I think it's fine as is personally